client/layout/sidebar/style.scss

```diff
@@ -241,6 +249,11 @@ form.sidebar__button {
 			fill: $white;
 		}
 
+		&.preview .gridicons-external {
```

Comment:
This & the next block should have been `.sidebar__external` instead of the gridicon class. Will update in the morning.